src/ucs/config/parser.h

```diff
@@ -346,7 +346,9 @@ ucs_config_parser_set_default_values(void *opts, ucs_config_field_t *fields);
  *
  * @param opts           User-defined options structure to fill.
  * @param fields         Array of fields which define how to parse.
- * @param env_prefix     Prefix to add to all environment variables.
+ * @param env_prefix     Prefix to add to all environment variables,
```

Comment:
should we rename this to 'prefix' as well?

Reply:
So if function does argument split inside then argument name is env_prefix, in contrast if function directly use argument as is without any splitting then argument name is prefix.
Following this logic argument name in ucs_config_parser_fill_opts should be env_prefix.
WDYT?

Reply:
ok